packages/appstore/src/components/main-title-bar/__tests__/account-type-dropdown.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import AccountTypeDropdown from '../account-type-dropdown';
+import { render, screen } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import userEvent from '@testing-library/user-event';
+
+let mock = mockStore({});
+
+const checkAccountTypeDropdownContainer = () => {
+    const wrapper = ({ children }: { children: JSX.Element }) => <StoreProvider store={mock}>{children}</StoreProvider>;
+
+    const { container } = render(<AccountTypeDropdown />, { wrapper });
+    expect(container).toBeInTheDocument();
+};
```

Comment:
https://github.com/binary-com/deriv-app/pull/8608#discussion_r1194536810

Reply:
Will amend it 🙏